distros/wildfly-standalone/pom.xml

```diff
@@ -0,0 +1,123 @@
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.hawkular.metrics</groupId>
+    <artifactId>hawkular-metrics-distros</artifactId>
+    <version>0.14.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>hawkular-metrics-wildfly-standalone</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Hawkular Metrics WildFly Standalone Distribution</name>
+  <description>A WildFly server that is configured to run Metrics. Cassandra must be installed separately.</description>
+
+  <properties>
+    <assembly.dir>${project.build.directory}/assembly</assembly.dir>
+    <wildfly.dist.zip.root.dir>wildfly-${version.org.wildfly}</wildfly.dist.zip.root.dir>
+    <hawkular.dist.zip.root.dir>${artifactId}/hawkular-metrics-${project.version}</hawkular.dist.zip.root.dir>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.hawkular.metrics</groupId>
+      <artifactId>hawkular-metrics-api-jaxrs</artifactId>
+      <version>${project.version}</version>
+      <type>war</type>
+    </dependency>
+    <dependency>
+      <groupId>org.wildfly</groupId>
+      <artifactId>wildfly-dist</artifactId>
+      <version>${version.org.wildfly}</version>
+      <type>zip</type>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-wildfly</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>unpack-dependencies</goal>
+            </goals>
+            <configuration>
+              <excludeArtifactIds>wildfly-jms-client-bom</excludeArtifactIds>
+              <outputDirectory>${project.build.directory}/wildfly</outputDirectory>
+              <includeArtifactIds>wildfly-dist</includeArtifactIds>
+            </configuration>
+          </execution>
+          <execution>
+           <id>copy-hawkular-dependencies</id>
+           <phase>prepare-package</phase>
+           <goals>
+             <goal>copy-dependencies</goal>
+           </goals>
+           <configuration>
+             <outputDirectory>${project.build.directory}/wildfly/${wildfly.dist.zip.root.dir}/standalone/deployments</outputDirectory>
+             <includeArtifactIds>hawkular-metrics-api-jaxrs</includeArtifactIds>
+             <overWriteIfNewer>true</overWriteIfNewer>
+             <excludeTransitive>true</excludeTransitive>
+           </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>2.5.4</version>
```

Comment:
Please remove `<version>2.5.4</version>` from here and add an explicit `groupId`. If you need to override the version 2.6 managed in hawkular-parent, please rather override the `version.maven-assembly-plugin` property with a proper comment why you need it.